collections-lists/FlatteningNestedCollectionInC#/FlatteningNestedCollectionInC#/Methods/Methods.cs

```diff
@@ -0,0 +1,55 @@
+﻿using FlatteningNestedCollection.Models;
+
+namespace FlatteningNestedCollection;
+
+public class Methods
```

Comment:
Rename this to more meaningful name instead of Methods

Reply:
Renamed it